subdl

```diff
@@ -89,6 +97,29 @@ def file_base(filename):
 def gunzipstr(zs):
     return gzip.open(io.BytesIO(zs)).read()
 
+def filterSubs(filename):
+    import pysrt
+    print("Parsing subs...")
+    subs = pysrt.open(filename)
+    print(" Length: %d" % len(subs))
+    i = 0
+    removed = 0
+    while i < len(subs):
```

Comment:
you should start iterating from end. thus you don't have to conditionally update index.
you can just do this instead of this `while` loop.

    for i in range(len(subs) - 1, -1, -1):
      for f in FILTERS:
        if re.search(f, subs[i].text, re.M | re.I):
          del subs[i]
          removed += 1
          break